winterfell/app.js

```diff
@@ -1,47 +1,60 @@
 #!/usr/bin/env node
+var Biscuit = require('./services/biscuit');
+var bunyan = require('bunyan');
+var Constants = require('./utils/constants');
+var documentRenderingConfig = require('./config/documentRendering');
 var express = require('express');
-var session = require('express-session');
 var fs = require('fs');
+var Log = require('./utils/logHelper');
 var path = require('path');
-var bodyParser = require('body-parser');
-var Constants = require('./utils/constants');
-var ENVIRONMENT = Constants.environment;
-var documentRenderingConfig = require('./config/documentRendering');
+var session = require('express-session');
 
+var ENVIRONMENT = Constants.environment;
 var environment = process.env.NODE_ENV || ENVIRONMENT.LOCAL;
-var Biscuit = require('./services/biscuit');
 
+// Initialize App
 var app = express();
-app.set('secretsFile', require('./config/biscuit')[environment]);
-var biscuit = new Biscuit(app);
-
 app.environment = environment;
 app.baseUrl = Constants.baseUrl[app.environment];
 
-// Setup biscuit keys
-function setupBiscuitKey(keyName) {
-  var yamlKey = Constants.biscuitKeys[keyName];
-  var secret = biscuit.get(environment + '::' + yamlKey);
-  app.set(keyName, secret);
-}
-
-setupBiscuitKey(Constants.KEY_LOB_API);
-setupBiscuitKey(Constants.KEY_IDME_CLIENT_ID);
-setupBiscuitKey(Constants.KEY_IDME_SECRET_ID);
+// Initialize Logger
+var log = bunyan.createLogger({
+  name: 'vetafi_' + environment,
+  streams: [{
+    path: '../logs/app.js' // todo: these logs need to be moved into the same level
+  }]
+});
+app.log = log;                                            // Main logger
+app.logApi = app.log.child({widget_type: 'api'});         // Logger for API request/responses
+app.log.info("New logger created.");
```

Comment:
Can we store the logger in a separate module and not attach it to the `app` object? This will allow us to log messages in modules that do not have easy access to the `app` instance (such as utilities/helper methods etc).